modules/admin-ui/src/main/java/org/opencastproject/adminui/endpoint/EndpointUtil.java

```diff
@@ -111,6 +111,11 @@ public String apply(Date date) {
    * @throws JsonCreationException
    */
   public static <T> JSONObject generateJSONObject(Map<String, T> list) throws JsonCreationException {
+
+    if (list == null) {
```

Comment:
The list being null is not okay here because then list.entrySet() will throw a NullpointerException. But you're right, one could return an empty json object instead of throwing the JsonCreationException. Not sure what makes more sense...

Reply:
Oh, I haven't looked at the code close enough: If it throw an exception before anyway, your change is an improvement. Let's just keep it.